depends/packages/qt.mk

```diff
@@ -163,6 +163,7 @@ $(package)_config_opts_mingw32 += -no-dbus
 $(package)_config_opts_mingw32 += -no-freetype
 $(package)_config_opts_mingw32 += -xplatform win32-g++
 $(package)_config_opts_mingw32 += "QMAKE_CFLAGS = '$($(package)_cflags) $($(package)_cppflags)'"
+$(package)_config_opts_mingw32 += "QMAKE_CXX = '$($(package)_cxx)'"
```

Comment:
I confirm this line fixes the Qt build issue with Mingw for me.

However, when I inspect `Makefile.Release` files created by `qmake` I see the following:

```
CC            = @echo compiling $< && x86_64-w64-mingw32-gcc
CXX           = @echo compiling $< && x86_64-w64-mingw32-g++-posix
```

I feel, we should do the same for CC so this won't explode in the future?

What do you think?

Reply:
Yes, I've noticed this as well when was working on this PR.

I'm not aware of difference between `x86_64-w64-mingw32-gcc-posix` and `x86_64-w64-mingw32-gcc-win32`.

Anyway, on the master branch we actually use the latter.  So it was not my intention to change the status quo in this PR.